platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/consensus/framework/validation/NoEventsLost.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.test.consensus.framework.validation;
+
+import com.swirlds.common.config.ConsensusConfig;
+import com.swirlds.common.crypto.Hash;
+import com.swirlds.platform.internal.EventImpl;
+import com.swirlds.platform.test.consensus.framework.ConsensusOutput;
+import com.swirlds.test.framework.config.TestConfigBuilder;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Assertions;
+
+@SuppressWarnings("unused") // issue tracked #6998
+public class NoEventsLost {
+    private static final ConsensusConfig CONFIG =
+            new TestConfigBuilder().getOrCreateConfig().getConfigData(ConsensusConfig.class);
+
```

Comment:
If this is a utility class, does it need a private default constructor?

Reply:
added